C#/cs01/15.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+namespace Brainchild.Curriculum.Solutions
+{
+    // ArrayList like class IntList
+    // IntList should have two methods
+    // Add(int num)
+    // Get(int index)
+    // Count Property
+    public class IntList
+    {
+        int count = -1;
+        public int[] array = new int[1];
```

Comment:
and we don't expose the whole array.please look at the comment below

Reply:
So the array as a whole shouldn't be available in Tests?